libs/auth/src/shell.ts

```diff
@@ -1,10 +1,13 @@
 import { Buffer } from 'buffer';
 import { spawn } from 'child_process';
+import { Stream } from 'stream';
 import { assert } from '@votingworks/basics';
 
 interface RunCommandOptions {
   /** A function to run on child process close, e.g. for cleanup */
   onClose?: () => Promise<void>;
+  /** Data to pipe to the child process's standard input */
+  stdin?: Stream;
```

Comment:
In case you're wondering why I've used the `Stream` type here and not the more specific `NodeJS.ReadableStream`, you'll see a few commits from now that I need to support using a `CombinedStream` (a la [combined-stream](https://github.com/felixge/node-combined-stream)), which extends `Stream` and not specifically `NodeJS.ReadableStream`.

Another option here would be `NodeJS.ReadableStream | CombinedStream`. Will think some more and potentially change.